app.py

```diff
@@ -334,6 +347,10 @@ def cancel_carpool(carpool_id):
     cancel_form = CancelCarpoolDriverForm()
     if cancel_form.validate_on_submit():
         if cancel_form.submit.data:
+            _email_carpool_cancelled(
+                carpool_id,
```

Comment:
Instead of passing in the `carpool_id`, can you pass in `carpool` so the emailer doesn't have to do another lookup?